db/migrate/20180604145622_add_title_to_push_notification.rb

```diff
@@ -0,0 +1,10 @@
+class AddTitleToPushNotification < ActiveRecord::Migration
+  def up
+    add_column :push_notifications, :title, :string, null: false, default: 'Title for existing records'
+    change_column_default(:push_notifications, :title, nil)
```

Comment:
Why this? We already set a default in the previous line, also we have a not null constrain.

Reply:
hehehe, this code is yours and was about to ask the same 
:D https://github.com/coopdevs/timeoverflow/pull/377/commits/8069d5173007b39cc6bf37aa02b64f47846f291e#r193443513

Reply:
THAT'S IMPOSSIBLE